config/config.go

```diff
@@ -460,6 +460,17 @@ func mergeEnvironmentVariables(c *AgentConfig) *AgentConfig {
 			log.Infof("overriding API endpoint from env")
 			c.APIEndpoints[0].Endpoint = u
 		}
+		if site := os.Getenv("DD_SITE"); site != "" {
```

Comment:
Thanks for pointing this out 👍  I will remove the `else if ` statement, however the first `if` statement (line 463) will be needed I think because the env name for the `process_dd_url` is not handled by the datadog-agent config package (not the same convention): https://github.com/DataDog/datadog-agent/pull/2412#discussion_r223436757

Reply:
Oh yeah, correct